src/chocolatey/infrastructure.app/nuget/NugetList.cs

```diff
@@ -48,12 +64,11 @@ public static IEnumerable<IPackage> GetPackages(ChocolateyConfiguration configur
                 results = results.Skip(configuration.ListCommand.PageSize * configuration.ListCommand.Page.Value).Take(configuration.ListCommand.PageSize);
             }
 
-            return results.OrderBy(p => p.Id)
-                        .AsEnumerable()
-                        .Where(PackageExtensions.IsListed)
-                        .Where(p => configuration.Prerelease || p.IsReleaseVersion())

```

Comment:
* File source - which you get with list -lo
* NuGet.org itself
* And maybe a myget source if you can get your hands on one.

Earlier when @Jaykul was working on some of this that was similar I could not get the results to match. It wasn't necessarily the count, although it was off as well. It was the order coming back from the query, it was off. https://github.com/chocolatey/choco/pull/224#discussion_r28621540 and https://github.com/chocolatey/choco/pull/224#issuecomment-116647818